src/main/java/com/fasterxml/jackson/databind/node/ArrayNode.java

```diff
@@ -30,6 +30,11 @@ public ArrayNode(JsonNodeFactory nf) {
         _children = new ArrayList<JsonNode>();
```

Comment:
I thought about chaining the old constructor call to call the new constructor and provide a reasonable default of size 10 (what the default implementation is for an `ArrayList`).